.arcconfig

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is this file required?

Reply:
Even I wasn;t sure about this, I though it might be required for some building purposes. It was created in the cherry-pick[](https://github.com/ayushSethi22/airflow/commit/76fe7ac595ce4b91f689d116775e2a2a130c180c) present in the commit id : 76fe7ac595ce4b91f689d116775e2a2a130c180c